src/Mpociot/ApiDoc/ApiDocGeneratorServiceProvider.php

```diff
@@ -42,4 +42,16 @@ public function register()
             'apidoc.update',
         ]);
     }
+
+    /**
+     * Return a fully qualified path to a given file relative
+     * to the storage directory
+     *
+     * @param string $path
+     * @return string
+     */
+    public function resource_path($path = '')
```

Comment:
I think you can use the helper `base_path()` instead of this function. This would avoid repeating functionality.